### ISSUE TITLE
Container resizing Issues

### ISSUE THREAD

**Issue by luiq54** [gilzoide/godot-dockable-container#13]
Resizing containers in a column or row is not intuitive/straightforward.

https://user-images.githubusercontent.com/51489635/181934906-d7fc02a7-1599-44c1-8ac5-602e0db59624.mp4

As you can see from this, trying to resize one container leads to size change in some other container. 
This is probably because of the tree-type nature of organizing containers. Can this be changed so that other containers are not affected when resizing ?

**Comment by gilzoide** [comment]
Yup, this is definitely about how the layout is structured (binary tree of Split nodes with Panel nodes as leaves).
And yeah, it can definitely be changed, the only thing is that the implementation will get more complex.
Managing non-binary trees, making sure all Panels have their minimum sizes respected, will require a bunch of logic and testing, but it's certainly doable! And the result would be much better for the end user experience.

Just as a side note, I won't be able to work on anything in the next couple weeks.
But feel free to do so, I'll be glad to help with anything I can!

**Comment by Lemonbrush** [comment]
@mystchonky hello, is there any progress?
